### PR TITLE
Updated prereq to standard 3.1 install include link.

### LINKED ISSUE #16450: Prerequisites link
Prerequisites links to .net core 3.0 sdk instead of 3.1.

with .net core 3.0 end of support approaching on march 2020, linking to 3.1 would be better especially to the people who are unaware of 3.1.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ca6139d4-022c-395a-0abf-539660550df1
* Version Independent ID: 3dd23f8a-300e-3f78-1dc5-8b759aabdccc
* Content: [Get started with ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/getting-started/?view=aspnetcore-3.1&tabs=windows#feedback)
* Content Source: [aspnetcore/getting-started/index.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/getting-started/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-getstarted**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by wadepickett**
[Internal Review](https://review.docs.microsoft.com/en-us/aspnet/core/getting-started/index?view=aspnetcore-1.0&branch=pr-en-us-16453)

Fixes #16450 

Updated prereq section to use standard 3.1 install include link and retested the topic instructions with 3.1.

### CODE REVIEW

**Comment by wadepickett** [comment]
@Rick-Anderson, could you give me a suggestion to commit so I can test the commit suggestion feature?  Perhaps change the date meta data to yesterday.  Thanks for the help.

File: aspnetcore/getting-started/index.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ author: rick-anderson
 description: A short tutorial that creates and runs a basic Hello World app using ASP.NET Core. 
 ms.author: riande
 ms.custom: mvc
-ms.date: 09/22/2019
+ms.date: 01/07/2020
```
Comment (Rick-Anderson) [inline aspnetcore/getting-started/index.md]: ms.date: 1/7/2020
```suggestion
ms.date: 01/07/2020
```

Comment (wadepickett) [reply aspnetcore/getting-started/index.md]: Thanks.  Commit suggestion available but still grayed out.   I will set to ready for review just to see if that makes any difference.

Comment (wadepickett) [reply aspnetcore/getting-started/index.md]: Still grayed out.

Comment (Rick-Anderson) [reply aspnetcore/getting-started/index.md]: Can you unresolve this so GitHub can see it?

Comment (wadepickett) [reply aspnetcore/getting-started/index.md]: unresolved.

**Pull request merged**